packages/PriceLockup/__tests__/PriceLockup.spec.jsx

```diff
@@ -0,0 +1,129 @@
+      style: { color: 'hotpink' },
+    })
+    expect(priceLockup).not.toHaveProp('className', 'my-custom-class')
+    expect(priceLockup).not.toHaveProp('style')
+  })
+
+  describe('size prop is small', () => {
+    const priceLockup = doShallow({ size: 'small' })
+    it('will make price h2 sized', () => {
+      expect(priceLockup.html()).toContain('priceValueSignsmall')
+    })
+    it('will make rate text medium sized', () => {
+      priceLockup.setProps({ rateText: '/month' })
+      expect(priceLockup).toContainReact(<Text size="medium">/month</Text>)
+    })
+    it('will make top text small sized', () => {
+      priceLockup.setProps({ topText: 'top text' })
+      expect(priceLockup).toContainReact(<Text size="small">top text</Text>)
+    })
+  })
+
+  describe('size prop is medium', () => {
+    const priceLockup = doShallow({ size: 'medium' })
+    it('will make price h2 sized', () => {
+      expect(priceLockup.html()).toContain('priceValueSignmedium')
+    })
+    it('will make rate text medium sized', () => {
+      priceLockup.setProps({ rateText: '/month' })
+      expect(priceLockup).toContainReact(<Text size="medium">/month</Text>)
+    })
+    it('will make top text small sized', () => {
+      priceLockup.setProps({ topText: 'top text' })
+      expect(priceLockup).toContainReact(<Text size="small">top text</Text>)
+    })
+    it('will make bottom text medium sized', () => {
+      priceLockup.setProps({ bottomText: 'bottom text' })
+      expect(priceLockup).toContainReact(<Text size="medium">bottom text</Text>)
+    })
+  })
+
+  describe('size prop is large', () => {
+    const priceLockup = doShallow({ size: 'large' })
+    it('will make DollarSign h1 sized', () => {
+      expect(priceLockup.html()).toContain('dollarSignH1Style')
+    })
+    it('will make price h2 sized', () => {
+      expect(priceLockup.html()).toContain('priceValueSignlarge')
+    })
+    it('will make rate text large sized', () => {
+      priceLockup.setProps({ rateText: '/month' })
+      expect(priceLockup).toContainReact(<Text size="large">/month</Text>)
+    })
+    it('will make top text large sized', () => {
+      priceLockup.setProps({ topText: 'top text' })
+      expect(priceLockup).toContainReact(<Text size="large">top text</Text>)
+    })
+  })
+
+  describe('signDirection prop', () => {
+    const priceLockup = doMount()
+    it('is left, and dollar sign will be left of price', () => {
+      priceLockup.setProps({ signDirection: 'left' })
+      expect(priceLockup.text()).toEqual('$25/month')
+    })
+    it('is right, and dollar sign will be right of price', () => {
+      priceLockup.setProps({ signDirection: 'right' })
+      expect(priceLockup.text()).toEqual('25$/month')
+    })
+    it('is undefined, and dollar sign will be left of price', () => {
+      priceLockup.setProps({ signDirection: undefined })
+      expect(priceLockup.text()).toEqual('$25/month')
+    })
+  })
+
+  describe('rateText prop', () => {
```

Comment:
In this group, you're only testing the size of rateText, which is already done above. I think you can remove these tests.

Reply:
done